src/lib/expansion/expansion-panel.ts

```diff
@@ -40,9 +40,6 @@ import {matExpansionAnimations} from './expansion-animations';
 import {MatExpansionPanelContent} from './expansion-panel-content';
 import {MAT_ACCORDION, MatAccordionBase} from './accordion-base';
 
-// TODO(devversion): workaround for https://github.com/angular/material2/issues/12760
-export const _CdkAccordionItem = CdkAccordionItem;
```

Comment:
does this one need the `ctorParameters` forwarding?

Reply:
It no longer needs the workaround because there is an explicit constructor for the `MatExpansionPanel` class now.